### PR TITLE
Style updates for themes (fixes #3)

### LINKED ISSUE #3: Update styles to align with docsify theme
Following up on #2, it would be great if the styles can be updated to better align with the docsify theme being used. Styles such as color, font-size, and margins should inherit those from the docsify whenever possible. Currently, the styles injected by docsify-pagination assume the theme is `vue.css`, which will not always be the case.

Suggestions:

- Replace fixed unit values (e.g. `px`) with relative units (e.g. `em`, `rem`, and `%`)
- Remove hard-coded colors and rely on `currentColor` or `inherit` when possible
- Use inline `<svg>` icons (instead of background images) so they can be styled via CSS

As a test, try viewing the `example` site using docsify's `dark.css` theme.

Thanks!

### PR DESCRIPTION
**Pull request by jhildenbiddle**
Styles:

- Replaced fixed `color` values with `currentColor`
- Replaced fixed `font-size` values with relative units
- Replaced fixed `margin` values with relative units
- Replaced top margin `border` color with rgba value
- Replaced background svg icons with inline svg elements to allow styling via CSS
- Replaced svg `<path>` with `<polyline>` to allow styling using stroke properties
- Moved "themeable" svg styles to CSS (`stroke`, `stroke-width`, etc.)
- Fixed opacity on "previous" pagination item when a "Next" link is not present (should be full opacity when a "Next" link is not present)
- Fixed `:hover` effect on "previous" link (both `pagination-item-label` and `pagination-item-title` now respond to hover state)

Markup:

- Changed icon position (icons now "outside" rather than "inside" of previous/next text).
- Updated logic in `pagination.js` to only render `div.pagination-item` elements if a link will be displayed. This allows styling items based on the number of `div.pagination-item` elements present.
- Added additional theme (`dark.css`) and `<style>` for testing theme compatibility. These are commented out by default. Uncomment to test.

JavaScript:

- Fixed error when $docsify is not defined

Misc:

- Removed `postcss-inline-svg` dependency
- Removed `postcss-svgo` dependency
- Removed `uglify-js` dependency (no need to uglify/minify since CSS is inject on the client)
- Removed `postcss.config.js` (not needed since no plugins are used)

Testing:

1. Enable `dark.css` theme in `index.html`. Notice how all pagination elements adopt styles from the theme.
1. Enable individual rules in the `<style>` block. Notice how theme authors can now easily modify styles for their theme, including SVG properties.

Happy to answer any questions you may have, and thanks for reviewing the PR!

### CODE REVIEW

**Comment by jhildenbiddle** [comment]
FYI -- I should mention that I am a day or so away from releasing a theme system for docsify, hence the interest in making sure all plugins play nicely with various themes. I am particularly fond of `docsify-pagination`, and I look forward to including it as part of my project documentation.

Thanks!

File: postcss.config.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-module.exports = {
-  plugins: [
-    require('postcss-inline-svg'),
-    require('cssnano'),
```
Comment (imyelo) [inline postcss.config.js]: **cssnano** reduces the size of the ``docsify-pagination.min.js`` file about 3kb. So I'll keep it.

File: package.json [hunk 2]
```diff
@@ -55,7 +53,6 @@
     "rollup-plugin-commonjs": "^8.2.6",
     "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-resolve": "^3.0.0",
-    "rollup-plugin-postcss": "^1.5.1",
-    "uglify-js": "^3.2.2"
```
[the remark this answers is not in the document]
Comment (jhildenbiddle) [reply package.json]: Ahh yes. Looks like in my haste to push the PR I mixed up minifying JS with CSS. 🤦‍♂️

File: postcss.config.js (hunk 1 above)
Comment (jhildenbiddle) [reply postcss.config.js]: Sounds good. I didn't check to see the size different. I assumed (incorrectly) that it would negligible.

**Pull request merged**